### ISSUE TITLE
Jsonlog -  syslog facility

### ISSUE THREAD

**Issue by gpolaert** [michaelpq/pg_plugins#10]
Hi Michael,

I'm testing jsonlog formatter with postgres 9.5.
Actually, it's working pretty well but I have 2 questions:

1. How can I use jsonlog with syslog facility. Log are printed in Json to the stdout but the record is still using the standard formater with the syslog output.
2. On the stdout, I've some character before the Json object. Where could they come from?
```
�▒t{"timestamp":"2016-03-17 12:43:24.509C","pid":536,"session_id":"56eaa66c.218","error_severity":"LOG","message":"database system was shut down at 2016-03-17 12:43:23 UTC"}
�▒t{"timestamp":"2016-03-17 12:43:24.509C","pid":536,"session_id":"56eaa66c.218","error_severity":"LOG","message":"MultiXact member wraparound protections are now enabled"}
�t{"timestamp":"2016-03-17 12:43:24.509C","pid":526,"session_id":"56eaa66c.20e","error_severity":"LOG","message":"database system is ready to accept connections"}
�t{"timestamp":"2016-03-17 12:43:24.509C","pid":540,"session_id":"56eaa66c.21c","error_severity":"LOG","message":"autovacuum launcher started"}

```

**Comment by michaelpq** [comment]
In order to use syslog, you are going to need to patch this plugin I think, which would require you to mimic more or less write_syslog in elog.c. I am not planning to implement that, but that could be done easily I guess with a bit of time. Regarding the junk entries, that's something I was aware of that and I got too lazy to fix it... I guess that's now fixed with 9f192d10.

**Comment by gpolaert** [comment]
Actually, I'm reviewing this example and yours, plus elog.c. https://github.com/gleu/Hooks-in-PostgreSQL/blob/master/examples/loggingperdbname/loggingperdbname.c
I've right, I have to patch it in order to send logs to syslog. Thanks for  9f192d1 commit :).

**Comment by michaelpq** [comment]
Actually I am reopening the issue. This broke the case where log_collector = on. I need to think about that more carefully.

**Comment by michaelpq** [comment]
Now fixed with 4cafe19a. When the logging_collector is enabled, there are still a couple of junk prefixes... But there is nothing I can do about that, this is controlled by a flag available only at postmaster level. This fixes the log output to stdout anyway, so that clearly helps.
